mecanum_base_node:
  names: [ "frontLeft", "backLeft", "frontRight", "backRight" ]
  families: [ "mecanumBase" ]
  publish_odom: true
